README.md

```diff
@@ -25,6 +25,34 @@ The following must be installed in order run Jekyll locally.
 6. Open <http://localhost:4000/> to view the site
 7. When finished hit `ctrl + c` in the terminal window to stop the server
 
+### API Documentation Development Setup
+
+The API documentation is generated from [sparkpost-api-documentation-DEV](https://github.com/SparkPost/sparkpost-api-documentation-DEV].  See the [README](https://github.com/SparkPost/sparkpost-api-documentation-DEV/blob/master/README.md) in that repo for details on that build process.
+
+For an easy dev setup:
+
+- Check out sparkpost-api-documentation-DEV
+- Check out sparkpost.github.io alongside
+- Start the API docs watch-and-build process:
+
+```bash
+cd sparkpost-api-documentation-DEV
+grunt staticDev
```

Comment:
This discussion is continued in the [sparkpost-api-documentation-DEV PR](https://github.com/SparkPost/sparkpost-api-documentation-DEV/pull/143).  tl;dr we might want to decouple production rendering through Jekyll from dev-time renders which use just the API docs tooling.